plugin/storage/integration/elasticsearch_test.go

```diff
@@ -130,18 +119,33 @@ func (*ESStorageIntegration) initializeESFactory(t *testing.T, allTagsAsFields b
 }
 
 func (s *ESStorageIntegration) initSpanstore(t *testing.T, allTagsAsFields bool) {
-	f := s.initializeESFactory(t, allTagsAsFields)
+	f := s.initializeESFactory(t, []string{
+		fmt.Sprintf("--es.num-shards=%v", 5),
+		fmt.Sprintf("--es.num-replicas=%v", 1),
+		fmt.Sprintf("--es.index-prefix=%v", indexPrefix),
+		fmt.Sprintf("--es.use-ilm=%v", false),
+		fmt.Sprintf("--es.service-cache-ttl=%v", 1*time.Second),
+		fmt.Sprintf("--es.tags-as-fields.all=%v", allTagsAsFields),
+		fmt.Sprintf("--es.bulk.actions=%v", 1),
+		fmt.Sprintf("--es.bulk.flush-interval=%v", time.Nanosecond),
+	}, es.NewFactory)
+	af := s.initializeESFactory(t, []string{
+		"--es-archive.enabled=true",
+		fmt.Sprintf("--es-archive.tags-as-fields.all=%v", allTagsAsFields),
+		fmt.Sprintf("--es-archive.index-prefix=%v", indexPrefix),
```

Comment:
this can be dropped, for inheritance?

Reply:
@yurishkuro I don't think it can currently because the tests initialize the factory directly instead of going through the meta-factory (https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/integration/elasticsearch_test.go#L123-L124).

Reply:
why don't we go through meta-factory here? It would better approximate how v1 binary works.

Reply:
@yurishkuro We can do that - do we want that to be a blocker for this PR?

Reply:
I would like to have a verification that workaround is available, and have that in the description before we merge.